ccmlib/node.py

```diff
@@ -636,31 +636,36 @@ def wait_for_binary_interface(self, **kwargs):
         log for 'Starting listening for CQL clients' before checking for the
         interface to be listening.
 
-        Emits a warning if not listening after NODE_WAIT_TIMEOUT_IN_SECS seconds.
+        Emits a warning if not listening after given timeout (default NODE_WAIT_TIMEOUT_IN_SECS) in seconds.
         """
+        timeout = kwargs.get('timeout', NODE_WAIT_TIMEOUT_IN_SECS)
```

Comment:
because of
https://github.com/riptano/ccm/blob/484476494bda6d71f895826358722a7b1c47a3cf/ccmlib/node.py#L850